.remarkrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
A metadocumentation ? :)
In Atom, the markdown editor automatically apply the rules from `.remarkrc`.
In sublimetext or any other editor, there should be a plugin.
At last, you can use [remark-lint](https://github.com/wooorm/remark-lint) from the command line